CONTRIBUTING.md

```diff
@@ -46,26 +45,40 @@ Help make reviewing easier by following these guidelines:
 
 ### Local setup
 
-You can use Docker (surprise) to build and serve the files locally. 
+You can use Docker (surprise) to build and serve the files locally.
 
-> [!IMPORTANT]  
+> [!IMPORTANT]
 > This requires Docker Desktop version **4.24** or later, or Docker Engine with Docker
 > Compose version [**2.22**](https://docs.docker.com/compose/file-watch/) or later.
 
-1. Clone the repository:
+1. Fork the docker/docs repository.
+
+   1. Go to the [docker/docs](https://github.com/docker/docs) repository.
+   2. In the top-right corner of the page, select **Fork**.
```

Comment:
Pretty sure we can add the link for forking the project here directly, instead of the instruction, or?